contents/blog/mixpanel-alternatives.md

```diff
@@ -0,0 +1,322 @@
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Roll out features safely; toggle features for cohorts or individuals</td>
+        </tr>
+        <tr>
+            <td><strong>Experiments</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>            
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Test changes and analyze their impact</td>
+        </tr>
+        <tr>
+            <td><strong>User surveys</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>            
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Ask users for qualitative feedback and gather responses</td>
+        </tr>
+        <tr>
+            <td><strong>Heatmaps</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>            
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Track where users click and why</td>
+        </tr>
+        <tr>
+            <td><strong>Open source</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>            
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Build your own apps and contribute code</td>
+        </tr>
+        <tr>
+            <td><strong>Autocapture</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>            
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Capture events without manual instrumentation</td>
+        </tr>
+        <tr>
+            <td><strong>Notebooks</strong></td>
+            <td className="text-center">Public beta</td>            
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Create shareable notebooks for analysis</td>
+        </tr>                           
+    </tbody>
+</table>
+</div>
+
+![posthog](../images/blog/mixpanel-alternatives/posthog.png)
+
+### How popular is PostHog?
+
+According to [data compiled by Jason Packer](https://www.linkedin.com/posts/jhpacker_just-updated-the-popularity-numbers-on-my-activity-7112462135120601088-YLdh/), an independent analytics consultant, _PostHog is the fastest growing Mixpanel alternative_ in the market,
+
+**PostHog** is deployed by 2,365 (0.2%) of the top 1 million websites in 2023, up +327% from 554 the previous year. **Mixpanel** is deployed by 5,218 (0.5%), down -17% from 6,295 the previous year.
+
+### Why do companies use PostHog?
+
+According to [reviews on G2](https://www.g2.com/products/posthog/reviews), companies use PostHog because:
```

Comment:
```suggestion
According to [reviews on G2](https://www.g2.com/products/posthog/reviews), companies trust PostHog because:
```

Always make it emotive, imho